### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by crandmck**
This copies all the information from the documentation into this README, with some formatting and wording improvements.

The docs will then use a script to then incorporate the README, so the info can just live here.
@rashmihunt @raymondfeng  PTAL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,101 +1,482 @@
...
 
-For full documentation, see the official [Documentation](http://loopback.io/doc/en/lb2/Oracle-connector.html).
+For more information, see the [LoopBack documentation](http://loopback.io/doc/en/lb3/Oracle-connector.html).
 
-**NOTE**: This connector is compatible with Node v.0.10.x, v.12.x, and iojs v1.6.2.  
+## Prerequisites
+
+**Node.js**: The Oracle connector is compatible with Node v.0.10.x, v.12.x, and later.  
+
+**MacOS**: The Oracle connector does not work on Mac OSX 10.11 (El Capitan) due to [SIP](https://en.wikipedia.org/wiki/System_Integrity_Protection) runtime protections and restrictions.
```
Comment (cjbj) [inline README.md]: If this is for the new version that [uses node-oracledb](https://github.com/strongloop/loopback-connector-oracle/blob/master/lib/oracle.js#L12) as the underlying DB access add-on, then it is incorrect to say it doesn't work on 10.11.  It does work. See https://github.com/oracle/node-oracledb/blob/master/INSTALL.md#instosx

File: README.md [hunk 2]
```diff
@@ -1,101 +1,482 @@
...
+Specifically, DYLD environment variables are now ignored; hence, rendering the Oracle connector inoperable due to the DYLD_LIBRARY_PATH dependency.
+For a workaround, see [How to Disable SIP in OS X El Capitan](http://osxdaily.com/2015/10/05/disable-rootless-system-integrity-protection-mac-os-x/).
+
+**Windows**: On 64-bit Windows systems, the Oracle connector runs only on 64-bit version of Node.js.
```
Comment (cjbj) [inline README.md]: The node-oracledb add-on works on 32 Windows.

File: README.md [hunk 3]
```diff
@@ -1,101 +1,482 @@
...
+
+**Oracle**: The Oracle connector requires Oracle 8.x - 12.x.
```
Comment (cjbj) [inline README.md]: The node-oracledb add-on requires Oracle client libraries 11.2 onwards.  These can connect to Oracle DB 9.2 onwards.

Comment (crandmck) [inline README.md]: @cjbj Thank you very much for your comments.  I've made all the changes you suggested and noted at the beginning of the readme that the connector is based on node-oracledb.

**Comment by crandmck** [comment]
@raymondfeng PTAL

File: README.md [hunk 4]
```diff
@@ -1,101 +1,480 @@
...
 
-* On Fedora/CentOS/RHEL
+On Fedora/CentOS/RHEL, get it with this command:
 
-        ﻿sudo yum install libaio
+```
+sudo yum install libaio
+```
 
+**IMPORTANT**: Ensure `c:\instantclient_12_1\vc10` comes before `c:\instantclient_12_1`.
```
Comment (cjbj) [inline README.md]: This isn't needed for node-oracledb.  Is it needed for anything else?

Comment (raymondfeng) [inline README.md]: No. The DLL dependency is only required by node-oracledb.

Comment (crandmck) [inline README.md]: OK, I removed that sentence. Thanks again @cjbj.
Unless there are more review comments, I'll merge this tomorrow.

**Pull request merged**